packages/sources/coingecko-test/src/crypto-utils.ts

```diff
@@ -1,18 +1,15 @@
-import { AxiosRequestConfig, AxiosResponse } from 'axios'
```

Comment:
I already did the work for this in #2222. Wanna get that in first, or is that redundant with this PR?

Reply:
I think we can get this one in first which is newer, and then I can rebase the other one and check if there are pending changes